code/addons/measure/preview.js

```diff
@@ -1 +1 @@
-export * from './dist/esm/preview';
+import './dist/preview';
```

Comment:
This should be `* as`, because `preview.ts` has exports: https://github.com/luciana-mendonca/storybook/blob/build/addon-measure-tsup/code/addons/measure/src/preview.tsx

Reply:
I've added the `export  *` back and tested on `examples/official-storybook`. The `addon` is working properly and the compiled files seems to be generated correctly. 

Please, let me know if there's a different solution.